.github/workflows/continous-integration-docker.yml

```diff
@@ -490,6 +490,8 @@ jobs:
       test_id: lwd-full-sync
```

Comment:
This CI job must pass before we merge this PR:
https://github.com/ZcashFoundation/zebra/actions/runs/4180390129